js/build.js

```diff
@@ -173,7 +173,14 @@
   $(document)
     .on('show.bs.collapse', Collapsible.SELECTORS.collapse, function(){
       // Immediately when the expand action is fired
-      Collapsible.prototype.toggleChevron($(this).attr('id'), true);
+      var id = $(this).attr('id');
+      var label = $('[data-toggle="collapse"][data-target="#' + id + '"]').text().trim();
+      Fliplet.Analytics.trackEvent({
```

Comment:
if possible I would always track events after any other action, to make sure the UI feedback happens before

Reply:
@squallstar Updated 👌